src/Containers/NoteForm/NoteForm.js

```diff
@@ -1,43 +1,74 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { setError } from '../../actions';
+import API from '../../utils/api';
 
 class NoteForm extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      ...this.props.note
+      id: this.props.id,
+      title: this.props.title,
+      issues: this.props.issues
     }
   }
 
-  handleChange = (e) => {
-    const index = this.state.issues.findIndex((issue) => issue.id == e.target.id)
-    console.log('that', index)
-    const issues = this.state.issues.slice()
-    issues[index].body = e.target.value
-    this.setState({ issues })
+  handleBodyChange = (e) => {
+    const index = this.state.issues.findIndex((issue) => issue.id == e.target.parentElement.parentElement.id);
+    const issues = this.state.issues.slice();
```

Comment:
Why did we have to change this to e.target.parentElement.parentElement.id?

Reply:
I moved the unique ids to the parent elements so that there is only one reference to it. It was getting confusing because we have both issue ids and note ids to keep track of.